src/System.Windows.Forms/tests/UnitTests/System/Windows/Forms/LinkLabel.LinkLabelAccessibleObjectTests.cs

```diff
@@ -65,13 +66,100 @@ public void LinkLabelAccessibleObject_GetPropertyValue_ControlType_IsExpected_Fo
+        [InlineData(4)]
+        [InlineData(10)]
+        public void LinkLabelAccessibleObject_GetChildCount_IsExpected(int linkCount)
+        {
+            using LinkLabel linkLabel = new();
+
+            for (int i = 0; i < linkCount; i++)
+            {
+                linkLabel.Links.Add(new());
+            }
+
+            int actual = linkLabel.AccessibilityObject.GetChildCount();
+
+            Assert.Equal(linkCount, actual);
+            Assert.False(linkLabel.IsHandleCreated);
+        }
+
+        [WinFormsTheory]
+        [InlineData(-1)]
+        [InlineData(0)]
+        [InlineData(1)]
+        [InlineData(15)]
+        [InlineData(27)]
+        [InlineData(38)]
+        [InlineData(49)]
+        [InlineData(1000)]
+        public void LinkLabelAccessibleObject_GetChild_IsExpected(int childIndex)
+        {
+            using LinkLabel linkLabel = new();
+
+            for (int index = 0; index < 50; index++)
+            {
+                linkLabel.Links.Add(new());
+            }
+
+            LinkAccessibleObject expected = 0 <= childIndex && childIndex < linkLabel.Links.Count
+                ? linkLabel.Links[childIndex].AccessibleObject
+                : null;
+
+            var actual = (LinkAccessibleObject)linkLabel.AccessibilityObject.GetChild(childIndex);
+
+            Assert.Equal(expected, actual);
+            Assert.False(linkLabel.IsHandleCreated);
+        }
+
+        [WinFormsTheory]
+        [InlineData(true)]
+        [InlineData(false)]
+        public void LinkLabelAccessibleObject_GetPropertyValue_Enabled_IsExpected(bool isEnabled)
+        {
+            using LinkLabel linkLabel = new();
+
+            linkLabel.Enabled = isEnabled;
+            bool actual = (bool)linkLabel.AccessibilityObject.GetPropertyValue(UiaCore.UIA.IsEnabledPropertyId);
+
+            Assert.Equal(isEnabled, actual);
+            Assert.False(linkLabel.IsHandleCreated);
+        }
+
+        [WinFormsTheory]
+        [InlineData((int)UiaCore.NavigateDirection.FirstChild)]
+        [InlineData((int)UiaCore.NavigateDirection.LastChild)]
+        public void LinkLabelAccessibleObject_FragmentNavigate_IsExpected_NavigateToChild(int direction)
+        {
+            using LinkLabel linkLabel = new();
+
+            for (int index = 0; index < 4; index++)
+            {
+                linkLabel.Links.Add(new());
+            }
+
+            LinkAccessibleObject expected = (UiaCore.NavigateDirection)direction switch
+            {
+                UiaCore.NavigateDirection.FirstChild => linkLabel.Links[0].AccessibleObject,
+                UiaCore.NavigateDirection.LastChild => linkLabel.Links[^1].AccessibleObject,
+                _ => null,
+            };
+
+            var actual = (LinkAccessibleObject)linkLabel.AccessibilityObject.FragmentNavigate((UiaCore.NavigateDirection)direction);
+
+            Assert.Equal(expected, actual);
+            Assert.False(linkLabel.IsHandleCreated);
+        }
```

Comment:
This test seems too complicated and not obvious to me. It may be worth to divide this test into two for "FirstChild" and "LastChild" directions.

```suggestion
	[WinFormsFact]
	public void LinkLabelAccessibleObject_FragmentNavigate_IsExpected_NavigateToFirstChild()
	{
		using LinkLabel linkLabel = new();
		AccessibleObject accessibleObject = linkLabel.AccessibilityObject;
	
		for (int index = 0; index < 4; index++)
		{
			linkLabel.Links.Add(new());
		}
	
		Assert.Equal(linkLabel.Links[0].AccessibleObject, accessibleObject.FragmentNavigate(NavigateDirection.FirstChild));
		Assert.False(linkLabel.IsHandleCreated);
	}
```